### PR TITLE
Migrate to default Forge structure

### PR DESCRIPTION
**Pull request by sunbreak1211**

### CODE REVIEW

File: deploy/FlapperDeploy.sol [hunk 1]
```diff
@@ -20,9 +20,9 @@ import "dss-interfaces/Interfaces.sol";
 import { ScriptTools } from "dss-test/ScriptTools.sol";
 
 import { FlapperInstance } from "./FlapperInstance.sol";
-import { FlapperUniV2 } from "src/FlapperUniV2.sol";
-import { FlapperMom } from "src/FlapperMom.sol";
-import { OracleWrapper } from "src/OracleWrapper.sol";
+import { FlapperUniV2 } from "../src/FlapperUniV2.sol";
+import { FlapperMom } from "../src/FlapperMom.sol";
+import { OracleWrapper } from "../src/OracleWrapper.sol";
```
Comment (oldchili) [inline deploy/FlapperDeploy.sol:25]:
```suggestion
import { FlapperUniV2 } from "src/FlapperUniV2.sol";
import { FlapperMom } from "src/FlapperMom.sol";
import { OracleWrapper } from "src/OracleWrapper.sol";
```

File: test/FlapperMom.t.sol [hunk 2]
```diff
@@ -19,12 +19,12 @@ pragma solidity ^0.8.16;
 import "forge-std/Test.sol";
 
 import { DssInstance, MCD } from "dss-test/MCD.sol";
-import { FlapperInstance } from "src/deploy/FlapperInstance.sol";
-import { FlapperDeploy } from "src/deploy/FlapperDeploy.sol";
-import { FlapperUniV2Config, FlapperInit } from "src/deploy/FlapperInit.sol";
+import { FlapperInstance } from "../deploy/FlapperInstance.sol";
+import { FlapperDeploy } from "../deploy/FlapperDeploy.sol";
+import { FlapperUniV2Config, FlapperInit } from "../deploy/FlapperInit.sol";
```
Comment (oldchili) [inline test/FlapperMom.t.sol:24]:
```suggestion
import { FlapperInstance } from "deploy/FlapperInstance.sol";
import { FlapperDeploy } from "deploy/FlapperDeploy.sol";
import { FlapperUniV2Config, FlapperInit } from "deploy/FlapperInit.sol";
```

File: test/FlapperUniV2.t.sol [hunk 3]
```diff
@@ -17,13 +17,13 @@
 pragma solidity ^0.8.16;
 
 import "dss-test/DssTest.sol";
-import { FlapperUniV2 } from "src/FlapperUniV2.sol";
-import "src/tests/helpers/UniswapV2Library.sol";
+import { FlapperUniV2 } from "../src/FlapperUniV2.sol";
```
Comment (oldchili) [inline test/FlapperUniV2.t.sol:20]:
```suggestion
import { FlapperUniV2 } from "src/FlapperUniV2.sol";
```

File: test/FlapperUniV2.t.sol [hunk 4]
```diff
@@ -17,13 +17,13 @@
...
+import "./helpers/UniswapV2Library.sol";
 
 import { DssInstance, MCD } from "dss-test/MCD.sol";
-import { FlapperInstance } from "src/deploy/FlapperInstance.sol";
-import { FlapperDeploy } from "src/deploy/FlapperDeploy.sol";
-import { FlapperUniV2Config, FlapperInit } from "src/deploy/FlapperInit.sol";
+import { FlapperInstance } from "../deploy/FlapperInstance.sol";
+import { FlapperDeploy } from "../deploy/FlapperDeploy.sol";
```
Comment (oldchili) [inline test/FlapperUniV2.t.sol:25]:
```suggestion
import { FlapperInstance } from "deploy/FlapperInstance.sol";
import { FlapperDeploy } from "deploy/FlapperDeploy.sol";
```

File: test/FlapperUniV2.t.sol [hunk 5]
```diff
@@ -17,13 +17,13 @@
...
+import { FlapperUniV2Config, FlapperInit } from "../deploy/FlapperInit.sol";
```
Comment (oldchili) [inline test/FlapperUniV2.t.sol:26]:
```suggestion
import { FlapperUniV2Config, FlapperInit } from "deploy/FlapperInit.sol";
```

File: test/OracleWrapper.t.sol [hunk 6]
```diff
@@ -19,9 +19,9 @@ pragma solidity ^0.8.16;
 import "forge-std/Test.sol";
 
 import { DssInstance, MCD } from "dss-test/MCD.sol";
-import { FlapperDeploy } from "src/deploy/FlapperDeploy.sol";
-import { FlapperInit } from "src/deploy/FlapperInit.sol";
-import { OracleWrapper } from "src/OracleWrapper.sol";
+import { FlapperDeploy } from "../deploy/FlapperDeploy.sol";
+import { FlapperInit } from "../deploy/FlapperInit.sol";
+import { OracleWrapper } from "../src/OracleWrapper.sol";
```
Comment (oldchili) [inline test/OracleWrapper.t.sol:24]:
```suggestion
import { FlapperDeploy } from "deploy/FlapperDeploy.sol";
import { FlapperInit } from "deploy/FlapperInit.sol";
import { OracleWrapper } from "src/OracleWrapper.sol";
```

File: test/FlapperMom.t.sol [hunk 7]
```diff
@@ -19,12 +19,12 @@ pragma solidity ^0.8.16;
...
 
-import { FlapperMom } from "src/FlapperMom.sol";
-import { FlapperUniV2 } from "src/FlapperUniV2.sol";
+import { FlapperMom } from "../src/FlapperMom.sol";
+import { FlapperUniV2 } from "../src/FlapperUniV2.sol";
```
Comment (oldchili) [inline test/FlapperMom.t.sol:27]:
```suggestion
import { FlapperMom } from "src/FlapperMom.sol";
import { FlapperUniV2 } from "src/FlapperUniV2.sol";
```

**Review by oldchili — COMMENTED** [review]
Changes look good.
I prefer to keep the non relative paths, see suggestions.

**Comment by sunbreak1211** [comment]
@oldchili suggestions applied, will wait for a final check from you and the review @telome and then I'll merge

**Review by oldchili — APPROVED** [review]
LGTM

**Review by telome — APPROVED** [review]
LGTM

**Pull request merged**